packages/components/modal/src/modal.css

```diff
@@ -0,0 +1,76 @@
+/* General........................................................ */
+
+:host {
+}
+
+.container {
+	background-color: var(--ts-color-white);
+	box-shadow: var(--ts-boxshadow-border-default);
+	top: var(--ts-unit);
+	bottom: var(--ts-unit);
+	right: var(--ts-unit);
+	left: var(--ts-unit);
+	display: flex;
+	flex-direction: column;
+	position: absolute;
+	z-index: var(--ts-zindex-dialog);
+}
+
+header {
```

Comment:
You can reuse `ts-header`, you don't have to do the css for the header and title again

Reply:
Good point. Added a `ts-header`.